src/workloads/execution/SinusoidalReadWrites.yml

```diff
@@ -0,0 +1,123 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/server-execution"
+Description: |
+  This workload attempts to create a sinusoidal distribution of reads/writes to test
+  the behavior of execution control when the load on the system changes over time.
+
+  We do this with two actors, one for writes and one for reads and spawn 5000 client
+  threads. Though the actors run concurrently, we use the sleep functionality
+  to effectively stop work on one actor while the other reaches its peak. We then
+  alternate for a couple repetitions.
+
+  In this workload, we try to ensure that reads and writes do not overlap much.
+Keywords:
+- sinusoidal
+- execution control
```

Comment:
Is it ok that this keyword is actually two words? Does it handle that?

Reply:
Yep, it can handle multiple spaces!